docs/resources/rf_stack.md

```diff
@@ -0,0 +1,194 @@
+
+* `description` - (Optional, String, ForceNew) Specifies the description of the resource stack, which contain maximum of
+  255 characters.  
+  Change this parameter will create a new resource.
+
+* `agency` - (Optional, String, ForceNew) Specifies the configuration of the agencies authorized to IAC.  
+  Change this parameter will create a new resource.
+  The [object](#stack_agency) structure is documented below.
+
+* `template_body` - (Optional, String) Specifies the HCL/JSON template content for deployment resources.
+  This parameter and `template_uri` are alternative and required if `vars_body` is set.
+
+* `vars_body` - (Optional, String) Specifies the variable content for deployment resources.
+  This parameter and `vars_uri` are alternative.
+
+* `template_uri` - (Optional, String) Specifies the OBS address where the HCL/JSON template archive (**.zip** file,
+  which contains all resource **.tf.json** script files to be deployed) or **.tf.json** file is located, which describes
+  the target status of the deployment resources.
+
+* `vars_uri` - (Optional, String) Specifies the OBS address where the variable (**.tfvars**) file corresponding to the
+  HCL/JSON template located, which describes the target status of the deployment resources.
+
+* `enable_auto_rollback` - (Optional, Bool, ForceNew) Specifies whether to enable automatic rollback.  
+  If enabled, the stack resources will rollback automatically to the last stable state with deployment failure.
+  The default value is **false**.
+  Change this parameter will create a new resource.
+
+* `enable_deletion_protection` - (Optional, Bool, ForceNew) Specifies whether to enable delete protection.  
+  The default value is **false**.
+  Change this parameter will create a new resource.
+
+<a name="stack_agency"></a>
+The `agency` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the name of IAM agency authorized to IAC account.  
+  Change this parameter will create a new resource.
+
+* `provider_name` - (Required, String, ForceNew) Specifies the name of the provider corresponding to the IAM agency.  
+  Change this parameter will create a new resource.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The stack ID.
+
+* `status` - The current status of the resource stack.
+
+* `created_at` - The creation time.
+
+* `updated_at` - The latest update time.
+
+## Timeouts
+
+This resource provides the following timeouts configuration options:
+
+* `create` - Default is 20 minutes.
+* `update` - Default is 20 minutes.
+* `delete` - Default is 5 minutes.
+
+For most HCL templates, the timeout parameters needs to be manually configured by the user to ensure that resources can
+be deployed successfully on the RF resource stack, e.g.
+
+```HCL
+resource "huaweicloud_rf_stack" "test" {
+  ...
+
+  timeouts {
+    create = "1h" // Such as the creation of GaussDB instances.
+    update = "1h"
+  }
```

Comment:
if this part is in the template body, add EOT around  it to make the example clear

Reply:
Actually, it is the timeout for resource stack, not the timeout for template resource.

Reply:
hah, seems we use tf in tf is a bit difficult for users, better to suggest use obs url.

Reply:
sure, obs URLs are more clear